build.gradle

```diff
@@ -38,10 +37,6 @@ configurations {
     // include mods
     modImplementation.extendsFrom(modInclude)
     include.extendsFrom(modInclude)
-
-    // include libraries
-    implementation.extendsFrom(library)
```

Comment:
non-minecraft dependencies should use `implementation`, switching them from `library` to `modInclude` uses `modImplementation` and adds a redundant remap step

Reply:
But If you use implementaion, the dependencies won't be added into the meteor jar

Reply:
> But If you use implementaion, the dependencies won't be added into the meteor jar

yes, that’s intended gradle behaviour. `include` is the configuration which adds the dependency into the jar